homeassistant/components/met/weather.py

```diff
@@ -161,7 +165,7 @@ def _update(self, *_):
         """Get the latest data from Met.no."""
         self._current_weather_data = self._weather_data.get_current_weather()
         time_zone = dt_util.DEFAULT_TIME_ZONE
-        self._forecast_data = self._weather_data.get_forecast(time_zone)
+        self._forecast_data = self._weather_data.get_forecast(time_zone, self._hourly)
```

Comment:
Won't this duplicate the api calls unnecessarily? We should consolidate the api calls if possible, preferably by using our data coordinator:

https://developers.home-assistant.io/docs/integration_fetching_data#coordinated-single-api-poll-for-data-for-all-entities